source/driver.cpp

```diff
@@ -131,6 +131,10 @@ void Driver::reading()
     ss1 << GlobalV::global_out_dir << GlobalV::global_in_card;
     read_input.write_parameters(PARAM, ss1.str());
 
+#ifdef __EXX
```

Comment:
there is no need to have 'exx' appears in driver.cpp

Reply:
I will solve it later

Reply:
I've moved this part into function `ReadInput::read_parameters`.